### PR TITLE
Adapt separate_usr_partition to use YuiRestClient

### PR DESCRIPTION
**Pull request by jknphy**
See [poo#78100](https://progress.opensuse.org/issues/78100)

Verification run: [separate_usr_partition](https://openqa.opensuse.org/tests/1491784#step/separate_usr_partition/15)

### CODE REVIEW

File: lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm [hunk 1]
```diff
@@ -37,17 +38,19 @@ sub new {
 sub init {
     my ($self, $args) = @_;
     $self->SUPER::init($args);
-    $self->{ExpertPartitionerPage}        = Installation::Partitioner::LibstorageNG::v4_3::ExpertPartitionerPage->new({app => YuiRestClient::get_app()});
-    $self->{ClonePartitionsDialog}        = Installation::Partitioner::LibstorageNG::v4_3::ClonePartitionsDialog->new({app => YuiRestClient::get_app()});
-    $self->{CreatePartitionTablePage}     = Installation::Partitioner::LibstorageNG::v4_3::CreatePartitionTablePage->new({app => YuiRestClient::get_app()});
-    $self->{DeletingCurrentDevicesDialog} = Installation::Partitioner::LibstorageNG::v4_3::DeletingCurrentDevicesDialog->new({app => YuiRestClient::get_app()});
```
Comment (rwx788) [inline lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm:43]: I'm not sure if we should unity all warnings in single dialog, or is it exactly same one?

File: lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerPage.pm [hunk 2]
```diff
@@ -119,4 +125,10 @@ sub select_volume_group {
     return $self->select_item_in_system_view_table('LVM Volume Groups|' . $vg);
 }
 
+sub select_device_in_table {
```
Comment (rwx788) [inline lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerPage.pm:128]: As per our discussion, could you please change this method so we could revert workaround once we get selection activating menu? selection of the row sub-item like `/dev/sda|sda` should already work.

File: lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm (hunk 1 above)
Comment (jknphy) [reply lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm:43]: while we don't check the content of the dialog I guess for now with one class should be enough. Or I can leave this as base class and then implement those two to not repeat the code, wdyt?

Comment (rwx788) [reply lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm:43]: Yeah, base class like pop-up would work, but we can also leave it for later, as we also see a need in some base class for wizard page not to repeat `press_next` method in each page class.

**Pull request merged**